ansible/retronas_create_dirs.yml

```diff
@@ -20,11 +20,9 @@
 
 - name: "{{ my_name }} - build systems directory layout (set fact)"
   set_fact:
-    link_list: "{{ link_list|default([]) }} + ['ln -sfT {{ retronas_path }}/{{ item.1.name|lower }}/{{ item.0.src }} {{ retronas_path }}/{{ item.1.name }}/{{ item.0.dest }};' ]"
+    link_list: "{{ link_list|default([]) + ['ln -sfT {{ retronas_path }}/{{ item.1.name|lower }}/{{ item.0.src }} {{ retronas_path }}/{{ item.1.name }}/{{ item.0.dest }};' ] }}"
```

Comment:
this results in link_list having undefined entries

```
fatal: [localhost]: FAILED! => {"msg": "The conditional check 'link_list is defined' failed. The error was: error while evaluating conditional (link_list is defined): ['ln -sfT /data/retronas/roms/nec/pcengine/iso /data/retronas/roms/nec/turbografxcd;', 'ln -sfT /data/retronas/roms/nintendo/famicom /data/retronas/roms/nintendo/nes;', 'ln -sfT /data/retronas/roms/nintendo/superfamicom /data/retronas/roms/nintendo/snes;', 'ln -sfT /data/retronas/roms/sega/megacd /data/retronas/roms/sega/segacd;', 'ln -sfT /data/retronas/roms/sega/megadrive /data/retronas/roms/sega/genesis;', 'ln -sfT {{ retronas_path }}/{{ item.1.name|lower }}/{{ item.0.src }} {{ retronas_path }}/{{ item.1.name }}/{{ item.0.dest }};']: 'item' is undefined\n\n
```

Reply:
you need to check this against a clean system using the install_mister playbook, bios. saves, savestate, etc directories are not created either which causes the mister playbook to fail with these changes

Reply:
should state this needs to work on bullseye as well as bookworm as our primary target is still rpios